packages/app-store/typeform/DESCRIPTION.md

```diff
@@ -1,6 +1,7 @@
 ---
 description: Adds a link to copy Typeform Redirect URL to integrate with Routing Forms.
 items:
+  - <iframe src="https://www.youtube.com/embed/vcV-N2gLPZc" frameborder="0" allowfullscreen></iframe>
```

Comment:
Instead of adding an HTML string directly(that we need to **dangerously** parse), I would recommend something like this
<img width="1072" alt="Screenshot 2023-02-24 at 1 04 08 PM" src="https://user-images.githubusercontent.com/1780212/221119203-d62af388-e404-45b2-9a0b-00ab61e31e52.png">
It is more structured.  We should then parse the items with a zod schema [here](https://github.com/calcom/cal.com/blob/1623190c07c7c4bd5c9f4de2d032fe4bcf94c7d2/apps/web/pages/apps/%5Bslug%5D/index.tsx#L89)

Reply:
frameborder 0 and allowfullscreen can be added directly while creating an iframe

Reply:
Thank you for the review @hariombalhara. We will make the required changes